components/searchable_channel_list.jsx

```diff
@@ -178,7 +179,8 @@ export default class SearchableChannelList extends React.Component {
                         id='searchChannelsTextbox'
                         ref='filter'
                         className='form-control filter-textbox'
-                        placeholder={localizeMessage('filtered_channels_list.search', 'Search channels')}
+                        placeholder={{id: 'filtered_channels_list.search', defaultMessage: 'Search channels'}}
```

Comment:
Because we don't have any way to infer this from the `mmjstool i18nextract` tool, we should use here something like `{id: t('filtered_channels_list.search'), defaultMessage: 'Search channels'}`.

Reply:
Could we add support to that somehow? How does the mobile app do it? It would be nice to minimize the use of `t` in cases like this where we do the same thing a lot

Reply:
We have to be aware that this problem exist, we only detect certain cases `localizeMessage` calls, `FormatedMessage` components and a small set of other cases. So, when we add one case of this type, we are removing translations from the list of translations. We have to add the translations check to the CI to be sure that you are properly updating your translations every single time.

I can add the new type, but anybody can, that is in the mattermost/mattermost-utilities repo.